<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>655</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>515</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>515</width>
    <height>699999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Psheprasham :D</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Привет из Польши</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditD"/>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1" alignment="Qt::AlignTop">
      <widget class="QLineEdit" name="lineEditC"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Введите значение e</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Введите значение b</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Введите значение d</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Введите значение c</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditB"/>
     </item>
     <item row="0" column="0" alignment="Qt::AlignTop">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Введите значение a</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QLineEdit" name="lineEditA"/>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEditE"/>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>70</y>
      <width>233</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="formulaLabel">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Выберите существующее выражение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>a/(b-c)*(d+e)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(a+b)*(c-d)/e</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>a-(b+c*d)/e</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>a/b-((c+d)*e)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>a*(b-c+d)/e</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(a+b)*(c-d)/e</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>a*(b-c)/(d+e)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>a/(b*(c+d))-e</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(a+(b/c-d))*e</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>a*(b+c)/(d-e)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>a-(b/c*(d+e))</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(a-b)/(c+d)*e</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>a/(b+c-d*e)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>a*(b-c)/(d+e)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(a+b*c)/(d-e)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="evaluate1">
         <property name="text">
          <string>Получить ответ</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="formulaLabel_2">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Введите своё выражение</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="formulaEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="evaluate2">
           <property name="text">
            <string>Получить ответ</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="kit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>281</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
